### PR TITLE
Feature/refactor

### LINKED ISSUE #45: Build functionality to assign tags to dropdown menu dynamically

### PR DESCRIPTION
**Pull request by oacortez**
What (if any) features are you implementing?
- Fix minor bugs for tags and nav ingredients filter

What (if anything) did you refactor?
- clean up code and remove unnecessary strings values

Were there any issues that arose?
- none

Is there anything that you need from your teammate?
- Please review my PR request

Any other comments, questions, or concerns?
- none

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -13,37 +13,16 @@ <h1 class="whats-cookin" id="whats-cookin">Whats Cookin</h1>
         <button class="nav-btn" id="all-recipes-btn">All Recipes</button>
         <button class="nav-btn" id="favorites-btn">Favorites</button>
         <button class="nav-btn" id="cookbook-btn">Cookbook</button>
-        <select name="tags" class="nav-btn" id="tags">
-          <option value="">Choose a Tag</option>
-          <option value="antipasti">Antipasti</option>
```
Comment (philalewis) [inline src/index.html:18]: Good to get rid of these

File: src/scripts.js [hunk 2]
```diff
@@ -52,22 +54,37 @@ function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
 
+function createDropdownTags() {
+  tags.innerHTML = '<option value="">Choose a Tag</option>'
```
Comment (philalewis) [inline src/scripts.js:58]: Good way to add tags to the dropdown with the JS

File: src/index.html [hunk 3]
```diff
@@ -13,37 +13,16 @@ <h1 class="whats-cookin" id="whats-cookin">Whats Cookin</h1>
         <button class="nav-btn" id="all-recipes-btn">All Recipes</button>
         <button class="nav-btn" id="favorites-btn">Favorites</button>
         <button class="nav-btn" id="cookbook-btn">Cookbook</button>
-        <select name="tags" class="nav-btn" id="tags">
```
Comment (mr-ryan12) [inline src/index.html:16]: Nice job removing the hard coded HTML and implementing functions to add tags dynamically in the scripts.js file.

**Review by philalewis — APPROVED** [review]
Looks good to me

File: src/scripts.js [hunk 4]
```diff
@@ -52,22 +54,37 @@ function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
 
+function createDropdownTags() {
```
Comment (mr-ryan12) [inline src/scripts.js:57]: Nice job building out this function with iterator methods. Good use of correct syntax.

**Pull request merged**